packages/tooltip/src/index.tsx

```diff
@@ -285,14 +290,74 @@ function useTooltip<T extends HTMLElement>({
     return () => ownerDocument.removeEventListener("keydown", listener);
   }, []);
 
-  function handleMouseEnter() {
+  React.useEffect(() => {
+    /*
+      This is a workaround for using tooltips with disabled controls in Safari.
+      Safari fires `pointerenter` but does not fire `pointerleave`
+      and `onPointerEventLeave` added to the trigger element will not work
+    */
+
+    if (!("PointerEvent" in window)) return;
+
+    let ownerDocument = getOwnerDocument(ownRef.current)!;
+
+    function listener(event: MouseEvent) {
+      // @ts-ignore `disabled` does not exist on HTMLDivElement but tooltips can be used with different elements
+      if (state !== VISIBLE || !ownRef.current?.disabled) return;
+
+      let target = event.target as Element | null;
+      if (
+        target?.hasAttribute("data-reach-tooltip-trigger") ||
+        target?.closest("[data-reach-tooltip-trigger][aria-describedby]")
```

Comment:
Can you explain the addition of `[aria-describedby]` in the selector here?

Reply:
We can have multiple disabled elements (icon buttons, for example) with tooltips next to each other, so I wanted to make sure that `GLOBAL_MOUSE_MOVE` will be triggered if you quickly move to the next disabled element. And I missed `aria-describedby` in the first part of the condition initially. 

At first, I wanted to use `ownRef.current?.isEqualNode(target) || ownRef.current?.contains(target)` instead, but that didn't work and I'm not sure why.

Reply:
Ah I think that makes sense. That could be a buggy assumption though seeing as how you can arbitrarily pass `aria-describedby` as well. I might add a data selector to the trigger params. We already use `data-state` elsewhere, so maybe:

```js
data-state={isVisible ? 'tooltip-visible' : 'tooltip-hidden'}
```

That might make the selector a bit more reliable! Also, you can just get away with `ownRef.current.contains` I think, that should return true even if the element itself matches the query.

Reply:
`ownRef.current.contains` doesn't work in this case, unfortunately. I assume because of ref mutations. 

But I like `data-state`, it's also more intuitive than the previous selector. Thanks!  Also, `closest` returns the element itself so I removed checks for attributes on the target.